spec/gemstash/gem_yanker_spec.rb

```diff
@@ -0,0 +1,157 @@
+require "spec_helper"
+
+describe Gemstash::GemYanker do
+  let(:auth_key) { "auth-key" }
+  let(:invalid_auth_key) { "invalid-auth-key" }
+  let(:auth_key_without_permission) { "auth-key-without-permission" }
+  let(:deps) { Gemstash::Dependencies.for_private }
+  let(:gem_name) { "example" }
+  let(:gem_version) { "0.1.0" }
+  let(:gem_slug) { "#{gem_version}-ruby" }
```

Comment:
Personally, I have mixed feelings about factories, but if it's something you'd like to tackle, I'm totally fine with that.